pkg/version/version_test.go

```diff
@@ -209,6 +209,33 @@ func TestNext(t *testing.T) {
 			want:    "",
 			wantErr: true,
 		},
+		{
+			name: "bump major not knowing minor and patch",
+			args: args{
+				current:          "1.5.2",
+				incrementPattern: "x.?.?",
```

Comment:
```suggestion
				incrementPattern: "x.0.0",
```

Reply:
Usually, the minor and patch levels are not carried over to the next major.

Reply:
My suggested change makes this test match the "increment major" test directly above. I would not add it in that case.